app/views/content/landing/how-to-become-a-teacher.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "How to become a teacher"
+description: A description goes here
```

Comment:
```suggestion
description: Our step by step guide for how to become a teacher. Find out more about checking your qualifications, how to find your training, and applying to train to be a teacher.
```

Reply:
Was just about to add a comment that the meta description needed adding :-) 

I know this is unlikely to ever be seen but do you mean 'fund your training' rather than 'find your training' @philspray ?

Reply:
Yes I mean fund not find